options/locale/locale_en-US.ini

```diff
@@ -1443,6 +1443,7 @@ issues.new.clear_assignees = Clear assignees
 issues.new.no_assignees = No Assignees
 issues.new.no_reviewers = No reviewers
 issues.new.blocked_user = Cannot create issue because you are blocked by the repository owner.
+issues.edit.already_changed = Unable to change issue content because issue is already changed
```

Comment:
any suggestion on this message? this also shown on pull request page.

Reply:
Maybe have another locale for pull request?

Reply:
done